src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -61,7 +60,7 @@ public class DurableTaskExtension :
         private readonly bool isOptionsConfigured;
         private IDurabilityProviderFactory durabilityProviderFactory;
         private INameResolver nameResolver;
-        private DurabilityProvider defaultDurabilityProvider;
+        private DurabilityProvider durabilityProvider;

```

Comment:
@ConnorMcMahon I hope you don't mind, but I decided to rename `defaultDurabilityProvider` to just `durabilityProvider`. I initially misinterpreted what this variable represented when I was investigating how to trace durability provider settings. Typically when I see that something is "default" then there must be some "non-default" version of it elsewhere. I looked for a non-default provider but could find one. What I eventually realized is that there is no "non-default" so I thought it would be more appropriate to remove the "default" prefix in this variable name to avoid future confusion.

Reply:
I don't have a huge complaint about it. The non-default `DurabilityProvider` in this case is the `DurabilityProvider` created in `DurableTask.Extension.GetClient()`, i.e. when the customer provides a different task hub name and/or connection string. I do think that distinction was moderately helpful, but I could definitely buy the argument that it adds more confusion than it helps, especially since those non-default `DurabilityProviders` are only referenced on a couple of lines.

Reply:
Oh, I see. So the way you were using it actually was in agreement with my definition of "default". I just failed to notice/remember the `GetClient()` case where a different instance gets created.

With that additional context, I think I'll roll this back to the old name. Thanks for letting me know about this.